package/MDAnalysis/coordinates/base.py

```diff
@@ -425,6 +420,15 @@ def __eq__(self, other):
             if not (self.positions == other.positions).all():
                 return False
 
+        if self.dimensions is None:
```

Comment:
Can we add tests for this?

Reply:
@richardjgowers if you don't want to add tests for this please open an issue for it.